Jenkinsfile.promote

```diff
@@ -152,20 +163,24 @@ boolean isDeployLatestTag(){
     return params.DEPLOY_WITH_LATEST_TAG
 }
 
-void loginOpenshiftRegistry() {
-    // Use creds & route as `jenkins-slave` is not running directly on Openshift
-    withCredentials([string(credentialsId: 'openshift-login-passwd', variable: 'OC_PASSWORD')]) {
-        sh "set +x && oc login -u admin -p ${OC_PASSWORD} --server=${OPENSHIFT_API} --insecure-skip-tls-verify"
-        sh "set +x && docker login -u admin -p \$(oc whoami -t) ${OPENSHIFT_REGISTRY_ROUTE}"
+void loginOpenshift(){
+    withCredentials([usernamePassword(credentialsId: "OPENSHIFT_CREDS", usernameVariable: 'OC_USER', passwordVariable: 'OC_PWD')]){
+        sh "oc login --username=${OC_USER} --password=${OC_PWD} --server=${OPENSHIFT_API} --insecure-skip-tls-verify"
     }
 }
 
+void loginOpenshiftRegistry(){
+    loginOpenshift()
+    // username can be anything. See https://docs.openshift.com/container-platform/4.4/registry/accessing-the-registry.html#registry-accessing-directly_accessing-the-registry
+    sh "set +x && podman login -u openshift -p \$(oc whoami -t) --tls-verify=false ${OPENSHIFT_REGISTRY}"
```

Comment:
can we remove this user from here and use envs?

Reply:
Like I wrote into the comment just before, the username can be anything it won't be taken into account.
If you want I can put `anything` instead of `openshift` as username but I don't see the point of overloading the env with an information which is useless in the end.